# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/parted-uclibc.patch
# Copyright (C) 2004 - 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- parted-1.8.1/libparted/labels/dasd.c.vanilla	2007-01-03 18:42:36.000000000 +0000
+++ parted-1.8.1/libparted/labels/dasd.c	2007-01-03 18:42:39.000000000 +0000
@@ -41,7 +41,6 @@
 #include <parted/fdasd.h>
 #include <parted/linux.h>
 
-#include <libintl.h>
 #if ENABLE_NLS
 #  define _(String) dgettext (PACKAGE, String)
 #else
--- parted-1.8.1/libparted/labels/fdasd.c.vanilla	2007-01-03 18:42:45.000000000 +0000
+++ parted-1.8.1/libparted/labels/fdasd.c	2007-01-03 18:42:51.000000000 +0000
@@ -21,7 +21,6 @@
 
 #include <parted/parted.h>
 
-#include <libintl.h>
 #if ENABLE_NLS
 #  define _(String) dgettext (PACKAGE, String)
 #else
--- parted-1.8.1/libparted/labels/vtoc.c.vanilla	2007-01-03 18:43:02.000000000 +0000
+++ parted-1.8.1/libparted/labels/vtoc.c	2007-01-03 18:43:05.000000000 +0000
@@ -11,7 +11,6 @@
 
 #include <parted/parted.h>
 
-#include <libintl.h>
 #if ENABLE_NLS
 #  define _(String) dgettext (PACKAGE, String)
 #else
